### DISCUSSION TITLE
essay/2024/04/01/songdo

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#201] [Comments]
# essay/2024/04/01/songdo

지난 주말에 GDG 송도 헬로월드 2024 컨퍼런스에서 발표를 하고 왔습니다.

https://jeho.page/essay/2024/04/01/songdo.html

**Comment by Jiwook00** [comment]
발표 너무 재밌게 잘 들었습니다!!

**Reply by BenjaminKim** [reply]
와... 자리에 계셨던 분이신가보네요. 들어주셔서 고맙습니다. 🙏

**Comment by voidFunctionReturn0** [comment]
구글 메인에 떠서 보고 왓습니다 ㅎㅎ
귀중한 경험 공유하주셔서 감사합니다~!

**Reply by BenjaminKim** [reply]
예전에 서버 운영 물어보셨는데... 발표에서 다뤄봤습니다. 😁
